nexus/tests/output/nexus_tags.txt

```diff
@@ -76,115 +78,98 @@ timeseries_schema_get                    /timeseries/schema
 
 API operations found with tag "organizations"
 OPERATION ID                             URL PATH
-organization_get_policy                  /organizations/{organization_name}/policy
-organization_put_policy                  /organizations/{organization_name}/policy
-organizations_delete_organization        /organizations/{organization_name}
-organizations_get                        /organizations
-organizations_get_organization           /organizations/{organization_name}
-organizations_post                       /organizations
-organizations_put_organization           /organizations/{organization_name}
+organization_create                      /organizations
+organization_delete                      /organizations/{organization_name}
+organization_list                        /organizations
+organization_policy_update               /organizations/{organization_name}/policy
+organization_policy_view                 /organizations/{organization_name}/policy
+organization_update                      /organizations/{organization_name}
+organization_view                        /organizations/{organization_name}
 
 API operations found with tag "policy"
 OPERATION ID                             URL PATH
-policy_get                               /policy
-policy_put                               /policy
+policy_update                            /policy
+policy_view                              /policy
 
 API operations found with tag "projects"
 OPERATION ID                             URL PATH
-organization_projects_delete_project     /organizations/{organization_name}/projects/{project_name}
-organization_projects_get                /organizations/{organization_name}/projects
-organization_projects_get_project        /organizations/{organization_name}/projects/{project_name}
-organization_projects_get_project_policy /organizations/{organization_name}/projects/{project_name}/policy
-organization_projects_post               /organizations/{organization_name}/projects
-organization_projects_put_project        /organizations/{organization_name}/projects/{project_name}
-organization_projects_put_project_policy /organizations/{organization_name}/projects/{project_name}/policy
-
-API operations found with tag "racks"
-OPERATION ID                             URL PATH
-hardware_racks_get                       /hardware/racks
-hardware_racks_get_rack                  /hardware/racks/{rack_id}
+project_create                           /organizations/{organization_name}/projects
+project_delete                           /organizations/{organization_name}/projects/{project_name}
+project_list                             /organizations/{organization_name}/projects
+project_update                           /organizations/{organization_name}/projects/{project_name}
+project_update_policy                    /organizations/{organization_name}/projects/{project_name}/policy
```

Comment:
I'm with Karen on the operation IDs as operation IDs, but your comment suggests this choice is (for you) more about determining what they will translate to in the CLI. I don't think `oxide update-policy` is clearly better than `oxide policy update` (I would lean toward consistency with whatever else we have), but I also don't have as clear a picture in mind as either of you of what the rules will be for translating from operation ID to CLI command.

Reply:
I generally like to keep actions (create, delete, attach, import... etc) as subcommands to the resource. This imo keeps the amount of commands at a reasonable amount when listing them as more endpoints are introduced, helps when navigating the `--help` output as a new user, as well as making autocomplete easier :)

This means I'll generally go for
```console
$ oxide project --help
USAGE:
    oxide project [OPTIONS] <SUBCOMMAND>

OPTIONS:
<...>

SUBCOMMANDS:
    create    Create a new project.
    delete    Delete project.
    edit      Edit project settings.
    help      Print this message or the help of the given subcommand(s)
    list      List projects.
    policy    Manage policies.
    view      View project.
<...>

$ oxide project policy --help
USAGE:
    oxide project policy [OPTIONS] <SUBCOMMAND>

OPTIONS:
<...>

SUBCOMMANDS:
    view      View a policy.
    update    Update a policy.
<...>
```

instead of
```console
$ oxide project --help
USAGE:
    oxide project [OPTIONS] <SUBCOMMAND>

OPTIONS:
<...>

SUBCOMMANDS:
    create        Create a new project.
    delete        Delete project.
    help          Print this message or the help of the given subcommand(s)
    list          List projects.
    update        Update project settings.
    update-policy Update a project policy
    view          View project.
    view-policy   View a project policy
<Some other commands for new endpoints that don't exist yet...>
```

Reply:
Making this change